chat-plugins/trivia.js

```diff
@@ -408,7 +389,14 @@ var Trivia = (function () {
 
 	return {
```

Comment:
I think I'd be better off following a similar approach to how a couple of the other chat plugins are set up, since this one clearly isn't working. It'd also save having to export ``Trivia`` at all.